filtered_topic_name: /skidpad_detection
inverse_flag: 0 #1 means inverse matrix else 0 means do not inverse ,inverse means car need turn right a little 
input_topic: /velodyne_points
no_ground_point_topic: /points_no_ground
ground_point_topic: /points_ground
all_points_topic: /all_points
clip_height: 2.0
# sensor_height: 0.135
sensor_height: 0.18
min_distance: 0.1
max_distance: 100.0 #
sensor_model: 32 #velodyne product type
num_iter: 3 #iteration num
num_lpr: 5 # seeds num
th_seeds: 0.03 # seeds to the plane threshold
th_dist: 0.03 # points to the plane threshold
road_type: 1 # {1:skidpad,2:acceleration,3:track_drive}
z_up: 0.7 # clip z up threshold
z_down: -1.0 #clip z down threshold
vis: 1

# vlp16
# str_range: "5,10,15,20"
# str_seg_distance: "0.18,0.35,0.53,0.6,0.6"

# # vlp32
str_range: "5,10,15,20"
str_seg_distance: "0.15,0.3,0.5,0.6,0.6"
length:
  circle2lidar: 15.0

